scientific/src/commonTest/kotlin/unit/TimeUnitTest.kt

```diff
@@ -163,4 +164,27 @@ class TimeUnitTest {
     fun timeFromWeightAndMassFlowRateTest() {
         assertEqualScientificValue(1(Hour), 2(Gram) / 2(Gram per Hour))
     }
+
+    @Test
+    fun timeFromKinematicViscosityAndAreaTest() {
```

Comment:
It would be great if that name could read like a math operation its testing.
If I read it right you're validating that "time from (division of) Area by Kinematic Viscosity"

Reply:
Good point, i'm on it